### ISSUE TITLE
Table with toolbar accessibility: toolbar disappears on focus

### ISSUE THREAD

**Issue by blaise2s** [IBM/carbon-components-angular#1242]
## Detailed description

This is a **BUG** with `<ibm-table-toolbar>` and `<ibm-table-toolbar-search>`. When you tab through a page that has a table with a toolbar, upon initial focus to the toolbar, the toolbar disappears. The toolbar only reappears after you tab through the items in the toolbar, in my case after three more tabs. What's worse, if you don't tab those there more times and click of jump elsewhere, the toolbar is still gone. It only reappears when you tap through the number of times for which you have focusable elements in the toolbar, and only then. This should not happen, the functionality I expect is that of which after the toolbar reappears... being able to tab through the toolbar without is disappearing. (Yes, I checked, this even happens here: https://angular.carbondesignsystem.com/?path=/story/components-table--with-toolbar) **Additionally:** the `<ibm-table-toolbar-search>` is not an element that's accessible by tabbing, you should be able to tab focus on the search and be able to start typing when it's in focus, but this is not the case as it is skipped over. I expect to be able to tab focus on it and search, and if I tabbed again to move on to the next focusable item in the toolbar

I am working in Chrome Version 81.0.4044.113

```
"@carbon/icons-angular": "^10.8.2",
"carbon-components": "^10.10.3",
"carbon-components-angular": "^3.49.4"
```

I work on IBM Clinical Development (ICD), this is not a pressing issue but should be resolved as soon as possible. 

## Steps to reproduce the issue

1. Pull https://github.com/blaise2s/carbon-sb
2. `cd carbon-sb`
3. `npm i`
4. `npm start`

## Additional information

* [Screen Recording 2020-04-30 at 5.02.07 PM.zip](https://github.com/IBM/carbon-components-angular/files/4561235/Screen.Recording.2020-04-30.at.5.02.07.PM.zip)
